elifepubmed/generate.py

```diff
@@ -364,6 +365,27 @@ def set_abstract(parent, poa_article, abstract_label_types):
         set_abstract_text(abstract_tag, '', '')
 
 
+def set_plain_language_summary(parent, article):
+    "set an OtherAbstract tag to include the digest as plain-language-summary"
+    tag_name = 'OtherAbstract'
+    if hasattr(article, 'digest') and article.digest:
+        # Pubmed allows <i> tags, not <italic> tags
+        tag_converted_digest = eautils.replace_tags(article.digest, 'italic', 'i')
+        tag_converted_digest = eautils.replace_tags(tag_converted_digest, 'bold', 'b')
+        tag_converted_digest = eautils.replace_tags(tag_converted_digest, 'underline', 'u')
+        tag_converted_digest = tag_converted_digest.replace('<p>', '').replace('</p>', '')
+        tag_converted_digest = etoolsutils.escape_unmatched_angle_brackets(
+            tag_converted_digest, utils.allowed_tags())
+        tagged_string = '<' + tag_name + '>' + tag_converted_digest + '</' + tag_name + '>'
+        reparsed = minidom.parseString(tagged_string.encode('utf-8'))
+        xmlio.append_minidom_xml_to_elementtree_xml(
+            parent, reparsed
+        )
+        # set attributes
+        parent[-1].set('Language', 'eng')
```

Comment:
weird, can this be set before adding the digest to the XML rather than later? That way what gets added is already complete?

Reply:
Probably, by adding these to the tag string prior to parsing it.

Reply:
These are passed to the function in `elifetools.xmlio` now, which handles adding attributes to tags when converting a string to a minidom element.